components/join-us-modal/join-us-modal.jsx

```diff
@@ -0,0 +1,326 @@
+          show={this.state.currentStep === 2}
+          heading="Add a short bio"
+          subhead="Tell other Network members a bit about yourself"
+        >
+          <Form
+            ref="step2Form"
+            fields={this.state.fields[`step2`]}
+            inlineErrors={true}
+            onUpdate={(evt, name, field, value) =>
+              this.handleFormUpdate(evt, name, field, value)
+            }
+          />
+        </Step>
+        <Step show={this.state.currentStep === 3} heading="Add a profile photo">
+          <Form
+            ref="step3Form"
+            fields={this.state.fields[`step3`]}
+            inlineErrors={true}
+            onUpdate={(evt, name, field, value) =>
+              this.handleFormUpdate(evt, name, field, value)
+            }
+          />
+        </Step>
+      </div>
+    );
+  }
+
+  renderConfirmation() {
+    return (
+      <div className="text-center">
+        <h2 className="h3-heading">Discover and Connect!</h2>
+        <p className="body-large">Thanks for signing up!</p>
+        <p>
+          Check out projects, connect with other members. Upload your projects
+          or add more to your profile.
+        </p>
+      </div>
+    );
+  }
+
+  renderButtons() {
+    let primaryAction = (
+      <div>
+        <button
+          className="btn btn-secondary"
+          onClick={() => this.handleNextBtnClick()}
+          disabled={this.state.submitting ? `disabled` : null}
+        >
+          Continue
+        </button>
+      </div>
+    );
+
+    let secondaryAction = this.state.currentStep !== 1 && (
+      <button
+        className="btn btn-link inline-link"
+        onClick={() => this.handleSkipBtnClick()}
+      >
+        I'll do this later
+      </button>
+    );
+
+    if (this.state.showConfirmation) {
+      primaryAction = (
+        <div>
+          <Link
+            to="/"
+            className="btn btn-primary"
+            onClick={() => this.closeModal()}
+          >
+            Explore
+          </Link>
+        </div>
+      );
+
+      secondaryAction = (
+        <Link to="/myprofile" onClick={() => this.closeModal()}>
+          Continue editing my profile
+        </Link>
+      );
+    }
+
+    return (
+      <div className="text-center mt-5">
+        <div className="mt-4">
+          {primaryAction}
+          <div className="secondary-action mt-4">{secondaryAction}</div>
+        </div>
+      </div>
+    );
+  }
+
+  render() {
+    if (!this.state.showModal) return null;
+
+    return (
+      <Modal
+        isOpen={this.state.showModal}
+        onRequestClose={() => this.closeModal()}
+        shouldCloseOnOverlayClick={false}
+        className="join-us-modal"
+        overlayClassName="join-us-modal-overlay"
+      >
+        <button className="btn btn-close" onClick={() => this.closeModal()}>
+          <span className="sr-only">Close modal</span>
+        </button>
+        {!this.state.showConfirmation && this.renderProgressDots()}
```

Comment:
This is two separate template instructions both conditional on `showConfirmation`, let's refactor this to a single templating call that generates the right content based on whether or not that state variable is set.